image.go

```diff
@@ -199,10 +207,43 @@ func (s *BaseImage) validate() error {
 	return nil
 }
 
+func (p *PostInstall) validate() error {
+	if p == nil {
+		return nil
+	}
+
+	var errs []error
+
+	if err := validateSymlinks(p.Symlinks); err != nil {
+		errs = append(errs, err)
+	}
+
+	return errors.Wrap(goerrors.Join(errs...), "symlink")
+}
+
 func (s *BaseImage) fillDefaults() {
 	fillDefaults(&s.Rootfs)
 }
 
+func (p *PostInstall) normalizeSymlinks() {
+	if p == nil {
+		return
+	}
+
+	// validation has already taken place
+	for oldpath := range p.Symlinks {
+		cfg := p.Symlinks[oldpath]
+		if cfg.Path == "" {
+			continue
+		}
+
+		cfg.Paths = append(cfg.Paths, cfg.Path)
+		cfg.Path = ""
+		sort.Strings(cfg.Paths)
```

Comment:
This looks like an unnecessary sort since it should only have one entry.
Also just to note, generally don't want to sort user provided arrays.

Reply:
```suggestion
```